cicd/pyinstaller/requirment.txt

```diff
@@ -26,3 +26,4 @@ cryptography==2.8
 python-crontab==2.5.1
 confluent-kafka==1.5.0
 netifaces==0.10.9
+tzlocal==2.1
```

Comment:
Why cant we use datetime.timezone ? 
https://docs.python.org/3/library/datetime.html

Reply:
I need local timezone info. Datetime does not give me that info.

Reply:
removed tzlocal, don't need local timezone , astimezone() with parameter defaults to localtimezone.